plugins/modules/zos_copy.py

```diff
@@ -1705,33 +1707,56 @@ def copy_to_pdse(
         existing_members = datasets.list_members(dest)  # fyi - this list includes aliases
         overwritten_members = []
         new_members = []
+        bulk_src_members = ""
+        result = dict()
 
         for src_member, destination_member in zip(src_members, dest_members):
             if destination_member in existing_members:
                 overwritten_members.append(destination_member)
             else:
                 new_members.append(destination_member)
-
+            bulk_src_members += "{0} ".format(src_member)
+
+        # Copy section
+        if src_ds_type == "USS" or self.asa_text or len(src_members) == 1:
+            """
+            USS -> MVS : Was kept on member by member basis bc file names longer that 8
```

Comment:
Minor: should be 'than' not 'that'

Reply:
hey thanks rich, did updated it and run the ansible certification tests to validate.

<img width="874" alt="Screenshot 2024-01-29 at 7 34 25 PM" src="https://github.com/ansible-collections/ibm_zos_core/assets/23223623/da123aaf-6f97-46b1-b35d-b22bfd9def9c">